.cargo/config

```diff
@@ -1,4 +0,0 @@
-
-[build]
-target-dir = "target/rust/"
-rustflags = []
```

Comment:
Why is this configuration removed?

If the location of built Rust artifacts is now `$rustProjectDir/target/...`, then [the line](https://github.com/enso-org/enso/pull/1121/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR814
) (and possibly some others?) needs to be updated:

```
    javaOptions += {
      val root = baseDirectory.value.getParentFile.getParentFile.getParentFile
      s"-Djava.library.path=$root/target/rust/debug"
    },
```

Reply:
Unintended.